app/src/main/java/com/example/lhi06/residue/AlbumReview.java

```diff
@@ -0,0 +1,42 @@
+package com.example.lhi06.residue;
+
+/**
+ * Created by lhi06 on 20/07/2017.
+ */
+
+public class AlbumReview {
+
+    Album thisAlbum;
+    int rating;
+    String review = "";
```

Comment:
because when i create the object i will be creating at the end of the review and if no one enters edit text i dont want it to equal null and crash the program